src/renderer/components/composer/EmojiAndStickerPicker.tsx

```diff
@@ -39,27 +38,30 @@ export const StickerDiv = (props: {
     <div className='sticker-pack'>
       <div className='title'>{stickerPackName}</div>
       <div className='container'>
-        {stickerPackImages.map((filePath, index) => {
-          return (
-            <div className='sticker' key={index}>
-              <img
-                src={filePath}
-                onClick={onClickSticker.bind(null, filePath)}
-              />
-            </div>
-          )
-        })}
+        {stickerPackImages.map((filePath, index) => (
+          <button
```

Comment:
should I add an aria-label for this?

Reply:
with the filename? or what did you have in mind here?

Reply:
don't have anything specific in mind, but yeah maybe something like "Send FILENAME" would work?